src/index.js

```diff
@@ -159,13 +159,20 @@ export function app(props, container) {
     }
   }
 
-  function removeElement(parent, element, props) {
+  function removeElement(parent, element, node) {
     function done() {
       parent.removeChild(element)
+      if (node.children) {
+        for (var i = 0; i < node.children.length; i++) {
+          if (typeof node.children[i] !== "string") {
+            removeElement(element, element.children[i], node.children[i])
```

Comment:
Also, with the [changes I attempted here](https://github.com/picodom/picodom/compare/master...mindplay-dk:destroy-event), I was specifically trying to avoid recursive removal of every individual element of a removed a subtree - mainly performance reasons. That's a lot of unncessary DOM operations. A single removal of the parent node would be preferable - only the destroy-notifications need to account for child element removals.